bridge/hardhat/script/verify.sh

```diff
@@ -0,0 +1,58 @@
+#bin/bash
+
+network_flag='bsctestnet'
+
+### npx hardhat --network bsctestnet verify --constructor-args ./hardhat/etherscan/arguments.js
+# --contract contracts/zeppelin/upgradable/proxy/TransparentUpgradeableProxy.sol:TransparentUpgradeableProxy
+# 0x8c1901c031Cdf42a846c0C422A3B5A2c943F4944
```

Comment:
I'd use [a multiline comment](https://stackoverflow.com/a/43158193/3922574) so this is clearer

Reply:
This was just like some comments at the code to help, but not necessary, and what you send it kind of create a doc, so it prints running the script, I will remove it

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4